docs/serving/configuration/config-defaults.md

```diff
@@ -30,319 +32,497 @@ data:
-
-**Default**: `"300"` (5 minutes)
+* **Global key:** `revision-timeout-seconds`
+* **Per-revision annotation key:** `serving.knative.dev/revision-timeout-seconds`
+* **Possible values:** integer
+* **Default:** `"300"` (5 minutes)
 
 **Example:**
-```yaml
-apiVersion:  v1
-kind:  ConfigMap
-metadata:
-  name:  config-defaults
-  namespace:  knative-serving
-data:
-  revision-timeout-seconds: "300" # 5 minutes
-```
 
-### Max Revision Timeout Seconds
+=== "Global (ConfigMap)"
+    ```yaml
+    apiVersion:  v1
+    kind:  ConfigMap
+    metadata:
+      name:  config-defaults
+      namespace:  knative-serving
+    data:
+      revision-timeout-seconds: "300"
+    ```
+
+=== "Per Revision"
+    ```yaml
+    apiVersion: serving.knative.dev/v1
+    kind: Service
+    metadata:
+      name: helloworld-go
+      namespace: default
+    spec:
+      template:
+        metadata:
+          annotations:
+            serving.knative.dev/revision-timeout-seconds: "300"
+        spec:
+          containers:
+            - image: gcr.io/knative-samples/helloworld-go
+    ```
+
+### Max revision timeout seconds
 {% raw %}
-max-revision-timeout-seconds contains the maximum number of
-seconds that can be used for revision-timeout-seconds.
-This value must be greater than or equal to revision-timeout-seconds.
-If omitted, the system default is used (600 seconds).
+The `max-revision-timeout-seconds` value determines the maximum number of seconds that can be used for `revision-timeout-seconds`. This value must be greater than or equal to `revision-timeout-seconds`. If omitted, the system default is used (600 seconds).
 
-If this value is increased, the activator's terminationGraceTimeSeconds
-should also be increased to prevent in-flight requests being disrupted.
+If this value is increased, the activator's `terminationGraceTimeSeconds` should also be increased to prevent in-flight requests from being disrupted.
 {% endraw %}
 
-**Key**: `max-revision-timeout-seconds`
-
-**Default**: `"600"` (10 minutes)
+* **Global key:** `max-revision-timeout-seconds`
+* **Per-revision annotation key:** `serving.knative.dev/max-revision-timeout-seconds`
+* **Possible values:** integer
+* **Default:** `"600"` (10 minutes)
 
 **Example:**
-```yaml
-apiVersion:  v1
-kind:  ConfigMap
-metadata:
-  name:  config-defaults
-  namespace:  knative-serving
-data:
-  max-revision-timeout-seconds: "600" # 10 minutes
-```
 
-### Revision Cpu Request
+=== "Global (ConfigMap)"
+    ```yaml
+    apiVersion:  v1
+    kind:  ConfigMap
+    metadata:
+      name:  config-defaults
+      namespace:  knative-serving
+    data:
+      max-revision-timeout-seconds: "600"
+    ```
+
+=== "Per Revision"
+    ```yaml
+    apiVersion: serving.knative.dev/v1
+    kind: Service
+    metadata:
+      name: helloworld-go
+      namespace: default
+    spec:
+      template:
+        metadata:
+          annotations:
+            serving.knative.dev/max-revision-timeout-seconds: "600"
```

Comment:
Likewise this isn't an annotation but an upper bound on `Spec.Timeout` on the `Revision.Spec`